gcode.c

```diff
@@ -1022,7 +1005,6 @@ uint8_t gc_execute_line(char *line)
    group 6 = {M6} (Tool change)
    group 7 = {G40, G41, G42} cutter radius compensation
    group 8 = {G43} tool length offset (But G43.1/G94 IS SUPPORTED)
-   group 8 = {*M7} enable mist coolant
```

Comment:
Yes, it's correct.  If you look here: http://dynomotion.com/Help/GCodeScreen/EMC_Handbook/node65.html, the modal group numbers are duplicated (one set for 'G' commands, one set for 'M' commands).